src/components/plugins/index.ts

```diff
@@ -0,0 +1,4 @@
+import DummyPlugin from './DummyPlugin/DummyPlugin';
```

Comment:
Same here we can directly create a component names Plugin which could be later used as this

Reply:
Please elaborate.

Reply:
instead of making two different files for making two dummies what we can do is just make a structure for the dummy and then pass the values there

Reply:
They are just examples.
1. One built the correct way
2. And a generic component

Both with work